tests/main/cgroup-tracking/task.yaml

```diff
@@ -0,0 +1,141 @@
+summary: Each snap app and hook is tracked via cgroups
+systems:
+    # Ubuntu 14.04's special version of systemd doesn't have StartTransientUnit API.
+    # This is covered in more depth in the sister cgroup-tracking-failure test.
+    - -ubuntu-14.04-*
+
+details: |
+    This test creates a snap process that suspends itself and ensures that it
+    placed into the appropriate hierarchy.
+environment:
+    USER/root: root
+    USER/test: test
+prepare: |
+    # This feature depends on the release-app-awareness feature
+    snap set core experimental.refresh-app-awareness=true
+
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local test-snapd-sh
+
+    tests.session -u "$USER" prepare
+
+restore: |
+    snap unset core experimental.refresh-app-awareness
+    tests.session -u "$USER" restore
+    rm -f /tmp/*.pid /tmp/*.stamp
+    if [ "$USER" = root ]; then
+        systemctl --user stop dbus.service || true
+    fi
+
+execute: |
+    # TODO: add a variant that uses a service
+    if [ "$USER" = test ] && "$TESTSTOOLS"/version-compare --strict "$(systemctl --version | head -n 1 | cut -d ' ' -f 2)" -lt 238; then
```

Comment:
It's probably worth listing the oldest systems we currently support in the spread suite that this part actually runs on.

Reply:
Done now.